scripts/suite.py

```diff
@@ -28,6 +28,7 @@
   <config_yaml>               Optional extra job yaml to include
   -s <suite>, --suite <suite>
                               The suite to schedule
+  --wait                      Block until the suite is finished
```

Comment:
The extra 30 minutes is like an assert to avoid waiting forever. It will only happen if something else is broken and prevents the jobs from being stopped after the expected timeout.